docs/binary.md

```diff
@@ -5,6 +5,26 @@
 You can download pre-built binaries for the Promscale
 Connector [on our release page](https://github.com/timescale/promscale/releases).
 
+Downlaod Promscale
+
+```
+curl -L -o promscale https://github.com/timescale/promscale/releases/download/<VERSION>/<PROMSCALE_DISTRIBUTION>
+```
+
+Grant executable permissions to Promscale:
+
+```
+chmod +x promscale
+```
+
+To deploy Promscale, run the following command:
+```
+./promscale --db-name <DBNAME> --db-password <DB-Password> --db-ssl-mode allow
+```
+Note that the flags `db-name` and `db-password` refer to the name and password of your TimescaleDB database.
+
+Further note that the command above is to deploy Promscale with SSL mode being allowed but not required. To deploy Promscale with SSL mode enabled, configure your TimescaleDB instance with ssl certificates and drop the `--db-ssl-mode` flag.  Promscale will then authenticate via SSL by default.
+
 We recommend installing the [promscale](https://github.com/timescale/promscale_extension/releases)
```

Comment:
I wonder how useful this is, given that promscale does that itself.

Reply:
I agree with this. Maybe @cevian can answer this.

Reply:
Promscale needs to be available in the database installation.